node/src/chain_spec.rs

```diff
@@ -784,6 +796,15 @@ fn foucoco_genesis(
 			replace_griefing_collateral: FixedU128::checked_from_rational(1, 10).unwrap(), // 10%
 		},
 		nomination: foucoco_runtime::NominationConfig { is_nomination_enabled: false },
+		dia_oracle: foucoco_runtime::DiaOracleConfig {
+			authorized_accounts: authorized_oracles,
+			supported_currencies: vec![foucoco_runtime::AssetId::new(
```

Comment:
specifying only one supported currency at the time that oracle_keys OracleConfig required 3 oracle keys 
- DOT
- USDC
- FeeEstimation

security pallet status will be offline (Oracle Offline error)
Now I figure out that we need hardcode `Key::FeeEstimation` inside DiaOracleAdapter implementation or hardcode 
the case for Key::FeeEstimation and retrieve value from spacewalk oracle storage.
Because there is no way to configure `Key::FeeEstimation` on DIA oracle provider (pallet side)
@ebma @b-yap

Reply:
You are right. That's a problem.

Reply:
Here is a ticket. 
Will remove Key::FeeEstimation at all.
No one need it.
https://github.com/pendulum-chain/spacewalk/issues/233